### PR TITLE
AWSLex Update models to latest

### PR DESCRIPTION
**Pull request by awsmobilesdk**
AWSLex Update models to latest

### CODE REVIEW

File: AWSLex.podspec [hunk 1]
```diff
@@ -18,42 +18,5 @@ Pod::Spec.new do |s|
   s.private_header_files = 'AWSLex/Bluefront/include/*.h'
   s.preserve_paths = 'AWSLex/Bluefront/include/*'
   s.vendored_libraries = 'AWSLex/Bluefront/libBlueAudioSourceiOS.a'
-  s.resource_bundle = { 'AWSLex' => 'AWSLex/Media.xcassets' }
```
Comment (royjit) [inline AWSLex.podspec:21]: Was this manually added before?

Comment (lawmicha) [reply AWSLex.podspec:21]: moving to draft, need to investigate previous AWSLex model updates and compare

Comment (palpatim) [reply AWSLex.podspec:21]: We'll investigate the required codegen changes, but in the meantime we'll merge this in.

**Review by palpatim — APPROVED** [review]
Manually edited podspec file to include .xcconfig build settings. Will investigate required codegen changes separately

**Pull request merged**